students/yujeong/users/views.py

```diff
@@ -1,5 +1,38 @@
 import json
+import re
 
 from unittest     import result
 from django.http  import JsonResponse
 from django.views import View
+from users.models import User
+
```

Comment:
1. from unittest import result 부분은 사용하지 않고 있으므로 삭제해 주세요.
2. from users.models import User는 pip로 설치한 모듈이 아니고 프로젝트 내부에 있는 모듈이므로 django 모듈을 import 하는 부분과 구분을 지어주면 좋을 것 같습니다.

```suggestion

from users.models import User

```